app/src/main/java/com/example/affirmations/ui/theme/Theme.kt

```diff
@@ -58,14 +53,6 @@ fun AffirmationsTheme(
         darkTheme -> DarkColorScheme
         else -> LightColorScheme
     }
-    val view = LocalView.current
-    if (!view.isInEditMode) {
-        SideEffect {
-            val window = (view.context as Activity).window
-            window.statusBarColor = colorScheme.background.toArgb()
-            WindowCompat.getInsetsController(window, view).isAppearanceLightStatusBars = !darkTheme
-        }
-    }
```

Comment:
As discussed lets keep this for all the apps, if you have not updated the codelabs